doc/jumpstart/utils.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import absolute_import
```

Comment:
Question: Do we need the legal comment at the top?

Reply:
Probably